src/applications/personalization/profile/components/ProfileMobileSubNav.jsx

```diff
@@ -23,7 +23,7 @@ const ProfileMobileSubNav = ({ isLOA3, isInMVI, routes }) => {
 
   // on first render, set the focus to the h4
   useEffect(() => {
-    focusElement('#mobile-subnav-header');
+    focusElement('#mobile-subnav-header-button');
```

Comment:
could you update the comment two lines above this?

Reply:
completed